### PR TITLE
Update all examples that start with "b" to use jest

### PR DESCRIPTION
**Pull request by bdemann**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -321,7 +321,7 @@ jobs:
             - if: ${{ needs.release-candidate-deploy.outputs.should_run_tests && !contains(github.head_ref, 'release--') && !(github.ref == 'refs/heads/main' && contains(github.event.head_commit.message, 'Merge pull request') && contains(github.event.head_commit.message, 'demergent-labs/release--')) }}
               shell: bash -l {0}
               working-directory: ${{ matrix.example_directories }}
-              run: AZLE_PROPTEST_NUM_RUNS=5 AZLE_PROPTEST_VERBOSE=true npm test
+              run: AZLE_PROPTEST_NUM_RUNS=5 AZLE_PROPTEST_VERBOSE=true AZLE_INTEGRATION_TEST_LINK_AZLE=true npm test
```
Comment (bdemann) [inline .github/workflows/test.yml:324]: @lastmjs we should discuss this as part of the review

File: examples/bytes/test/tests.ts [hunk 2]
```diff
@@ -1,80 +1,28 @@
...
-            test: async () => {
+export function getTests(bytes_canister: ActorSubclass<_SERVICE>): Test {
+    return () => {
+        describe.each([
+            'example_1_kb.txt',
+            'example_10_kb.txt',
+            'example_100_kb.txt',
+            'example_1000_kb.txt',
+            'example_2000_kb.txt'
+        ])('', (fileName) => {
```
Comment (bdemann) [inline examples/bytes/test/tests.ts:16]: we need to add a description here

File: .github/workflows/test.yml (hunk 1 above)
Comment (bdemann) [reply .github/workflows/test.yml:324]: "I'm not sure you can just do this generally, we need to handle the npm and repo tests. There are tests that run with Azle installed from npm during release, not from the repo, those must be honored correctly. We should discuss." - @lastmjs

File: examples/bytes/test/tests.ts (hunk 2 above)
Comment (bdemann) [reply examples/bytes/test/tests.ts]: done

File: .github/workflows/test.yml (hunk 1 above)
Comment (lastmjs) [reply .github/workflows/test.yml]: We are testing this now

**Review by lastmjs — CHANGES_REQUESTED** [review]

**Review by lastmjs — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (lastmjs) [reply .github/workflows/test.yml]: We will remove these logs in a subsequent pr, testing in a release ci/cd

**Review by lastmjs — APPROVED** [review]

**Pull request merged**